python/tskit/trees.py

```diff
@@ -4652,11 +4674,11 @@ def haplotypes(
     def variants(
         self,
         *,
-        as_bytes=False,
         samples=None,
         isolated_as_missing=None,
         alleles=None,
         impute_missing_data=None,
+        return_variant_copies=None,
```

Comment:
This parameter name feels clunky to me, and does clutter up client code a bit. What do you think about just having ``copy`` instead?  I feel like I'll be writing this a lot, so I want it to be easy:
```python
for var in ts.variants(copy=False):
    # do stuff with var
```

Reply:
Initially I wanted it to be a bit of a mouthful as you can make mistakes with it - but the same goes for `.trees` I guess. You're right that we'll be almost always using it.
https://github.com/tskit-dev/tskit/pull/2172/commits/7acf71690a2d177209975ce637419c7740306426